diff --git a/dist/chunks/rpc.-pEldfrD.js b/dist/chunks/rpc.-pEldfrD.js
index a48eb01..3280fb8 100644
--- a/dist/chunks/rpc.-pEldfrD.js
+++ b/dist/chunks/rpc.-pEldfrD.js
@@ -52,7 +52,8 @@ function createRuntimeRpc(options) {
 			if (functionName === "onUnhandledError") message += ` with "${args[0]?.message || args[0]}"`;
 			throw new Error(message);
 		},
-		...options
+		...options,
+		timeout: -1
 	}));
 	return {
 		rpc,
